ci-operator/config/openshift-pipelines/pipelines-service/openshift-pipelines-pipelines-service-main.yaml

```diff
@@ -0,0 +1,35 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+releases:
+  latest:
+    release:
+      architecture: amd64
+      channel: stable
+      version: "4.10"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: plnsvc-test
+  steps:
+    cluster_profile: aws
+    test:
+    - as: plnsvc-test
+      cli: latest
+      commands: ls;echo "onboarding pipelines-service repo"
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+    workflow: ipi-aws
```

Comment:
Did you consider using [cluster claim](https://docs.ci.openshift.org/docs/how-tos/cluster-claim/#use-the-cluster-pool-from-a-ci-job) instead of full cluster install?

Currently the whole CI job takes 50 minutes. With cluster claim you can save up to 30 minutes for each job.
[The example of using the cluster pool](https://github.com/openshift/release/blob/f1d1315d0ec4aa2fb7a9ec816958f8996602032f/ci-operator/config/redhat-appstudio/build-service/redhat-appstudio-build-service-main.yaml#L21-L60)

Reply:
thank you for your suggestion, I will take a look later